server/db/index.js

```diff
@@ -0,0 +1,19 @@
+const mongoose = require('mongoose');
+mongoose.connect('mongodb://localhost/fec', { useNewUrlParser: true });
+
+const MovieSchema = new mongoose.Schema({
+  movieId: {type: Number, unique: true},
+  movieName: String,
+  casts: Array
+});
+
+const Movie = mongoose.model('Movie', MovieSchema);
+
+var getCasts = (movieId, cb) => {
```

Comment:
Functionality looks great but this variable doesn't need an 's' at the end! getCast would be better ^^